project-base/webpack.config.js

```diff
@@ -47,16 +47,146 @@ Encore
         }
     }))
     .addPlugin(new CopyPlugin([
-        { from: 'web/bundles/fpjsformvalidator', to: '../../assets/js/bundles/fpjsformvalidator', force: true }
+        { from: 'web/bundles/fpjsformvalidator', to: '../../assets/js/bundles/fpjsformvalidator', force: true },
+        { from: 'web/bundles/shopsysframework', to: '../../web/assets/admin/images', force: true }
     ]))
 ;
 
+// Frontend Config
+const WebfontsGenerator = require('webfonts-generator');
```

Comment:
You should place all require calls at the top of the file.

Reply:
Fixed